dataset/components/bids_result/schemas.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2022 Indoc Research
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from datetime import datetime
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class DatasetVerifyForm(BaseModel):
+    dataset_code: str
+    type: str
+
+
+class BIDSResultSchema(BaseModel):
+    """General BIDS result schema."""
+
+    dataset_code: str
+    created_time: datetime
+    updated_time: datetime
+    validate_output: Dict[str, Any]
+
+    class Config:
+        orm_mode = True
+        json_encoders = {
+            datetime: lambda v: str(v.strftime('%Y-%m-%dT%H:%M:%S')),
```

Comment:
There's no need to call `str` here, `strftime` returns a string.
```suggestion
            datetime: lambda v: v.strftime('%Y-%m-%dT%H:%M:%S'),
```